cv32e40s/tests/cfg/pma_test_cfg_1.yaml

```diff
@@ -1,10 +1,15 @@
 name: pma_test_cfg_1
 description: PMA configuration for the PMA_TEST_CFG_1 test case
-compile_flags:
+compile_flags: >
    +define+PMA_TEST_CFG_1
    +define+ZBA_ZBB_ZBC_ZBS
-plusargs:
+plusargs: >
    +enable_pma=1
+   +enable_zca_extension=1
+   +enable_zcb_extension=1
+   +enable_zcmt_extension=1
```

Comment:
Shouldn't zcmt wait?

Reply:
Needs to be enabled in riscv-dv to generate correct illegal instructions; zcmt-instructions are never randomly inserted into a stream, only with directed streams, so this is safe to do (and necessary)